### DISCUSSION TITLE
Adding DatePicker as an option for dropdown

### DISCUSSION THREAD

**Discussion by ShilpaSharePoint** [microsoft-search/pnp-modern-search#2474] [Ideas]
Part of our solution we want to render Datepicker on click of dropdown similar to the attached image. we need this part of search filter. Making use of Fluent UI in SPFX React component, but it increases size of the container instead displaying as dropdown. Any references or workarounds will be helpful. kinldy help.

![datepickeron dropdown](https://user-images.githubusercontent.com/102510249/197113865-3ae1b25f-4ae1-40b3-a75e-f26d48aaa26a.png)

**Comment by wobba** [comment]
Closest you get today is using the filter web part with the Date range template. If not you have to develop a custom component to do this. Combining date range and date interval would make sense.

**Comment by ShilpaSharePoint** [comment]
We have added custom control by combining date range and date interval, but that is increasing the size of the search filter webpart. Please suggest us any workaround.
